modules/contentbox/modules/contentbox-admin/views/_components/ui/Icon.cfm

```diff
@@ -1,20 +1,47 @@
 <cfoutput>
     <!--- Icon name --->
-    <cfparam name="args.name" 	default="info">
+    <cfparam name="args.name" 	 default="info">
+    <!--- Icon Size --->
+    <cfparam name="args.size"    default="md">
 
-    <cbicon class="flex justify-center">
-        <svg fill="none" stroke="currentColor" stroke-width="1.5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+    <cbicon class="cbicon cbicon-#args.size#">
```

Comment:
@esmeace I would say the we keep it consistent with the other view components.  Example, `Alert` uses `<alert>` , `Toggle` is `<toggle>`.  So if we use `Icon`, then it should be `<icon>`

Reply:
Done.